.github/workflows/push.yml

```diff
@@ -146,6 +146,10 @@ jobs:
       MYSQL_TEST_HOST: mysql
       PDO_MYSQL_TEST_DSN: mysql:host=mysql;dbname=test
       PDO_MYSQL_TEST_HOST: mysql
+      PDO_FIREBIRD_TEST_DATABASE: test.fdb
+      PDO_FIREBIRD_TEST_DSN: firebird:dbname=localhost:test.fdb
```

Comment:
@nielsdos 
maybe

```suggestion
      PDO_FIREBIRD_TEST_DSN: firebird:dbname=firebird:test.fdb
```

Reply:
I think @SakiTakamachi is right. It's because of this line.

https://github.com/php/php-src/blob/d8cee06073a8761493dadbc598a99b43b2d0b5e4/.github/workflows/push.yml#L143-L144

* x64: Tests are run directly on the host specified in `runs-on`
* x32: A `ubuntu:20.04` container is created inside the `runs-on` host, and tests are run inside it.

I tried to run x32 tests directly in the runs-on container, but it seems very difficult.

So it seems best to fix it the way Saki says. In that case, please note that actions/test-linux and x64 tests will also need some adjustments.

Reply:
Most of the tests passed in [my fork](https://github.com/KentarouTakeda/php-src/pull/12/commits/062e6a1af8d78217bdf6aa2802125765b55a4675).

However, some tests failed because [they were written to reference `::1`](https://github.com/KentarouTakeda/php-src/actions/runs/11089157458/job/30809935544#step:9:161). I don't know how to fix these yet.

Reply:
Good finds! I missed this nuance completely, I'll import your commit. Thanks a lot @KentarouTakeda and @SakiTakamachi